ambry-clustermap/src/main/java/com.github.ambry.clustermap/AmbryReplica.java

```diff
@@ -86,18 +86,12 @@ public long getCapacityInBytes() {
     return capacityBytes;
   }
 
-  /**
-   * @return true if this replica is in sealed state.
-   */
-  boolean isSealed() {
+  @Override
+  public boolean isSealed() {
     return isSealed;
   }
 
-  /**
-   * Set the sealed state of this replica.
-   * @param isSealed true if the replica is to be set as sealed; false otherwise.
-   */
-  void setSealedState(boolean isSealed) {
+  public void setSealedState(boolean isSealed) {
```

Comment:
package private

Reply:
Actually there should be no changes from master here - the previous version (with the doc) needs to be retained

Reply:
changed